examples/bootstrap/example.tfvars

```diff
@@ -1,8 +1,15 @@
-resource_group_name  = "example-rg"
-location             = "East US"
-storage_account_name = "examplebootstrap"
+resource_group_name = "example-rg"
+location            = "West US"
+# storage_account_name       = "examplebootstrap"
```

Comment:
I assume this one should be uncommented?

Reply:
👍🏻